lib/knapsack_pro/config/ci/heroku.rb

```diff
@@ -0,0 +1,31 @@
+module KnapsackPro
+  module Config
+    module CI
+      class Heroku < Base
+        def node_total
+          # not provided
+        end
+
+        def node_index
+          # not provided
```

Comment:
Here we should be able to read from Heroku ENV the index of current parallel node. It should start from 0. So in case when we have 5 node total then the node_index is within range 0 - 4.

If Heroku won't expose this node index then we can manually set `KNAPSACK_PRO_CI_NODE_INDEX`